docusaurus/docs/dev-docs/installation/cli.md

```diff
@@ -89,6 +91,34 @@ Experimental Strapi versions are released every Tuesday through Saturday at midn
 Please use these experimental builds at your own risk. It is not recommended to use them in production.
 :::
 
+### Self-hosting your Strapi project
+
+When the installation script runs, the terminal will first ask you if you want to login/signup. Choosing `Login/signup` will create a free, 14-day trial [Strapi Cloud](/cloud/intro#what-is-strapi-cloud) account as described in the [Quick Start Guide](/dev-docs/quick-start).
```

Comment:
I think the wording is misleading.
Its not a 14-day trial "Cloud account", but a 14-day trial Project in a Cloud account.